scripts/generate-releases-manifest.mjs

```diff
@@ -14,10 +14,8 @@
 //       }
 //     },
 //     "win32": {
-//       "x64": {
-//         "sha": "abcdef1234567890",
-//         "url": "https://cdn.a8c-ci.services/studio/studio-win32-x64-abcdef1234567890.exe.zip"
-//       }
+//       "sha": "abcdef1234567890",
+//       "url": "https://cdn.a8c-ci.services/studio/studio-win32-abcdef1234567890.exe.zip"
```

Comment:
Looks good, thanks for fixing the example according the implementation.

I've added https://github.com/Automattic/dotcom-forge/issues/7317 to add support for Windows in the updates endpoint.